public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -2334,6 +2334,300 @@ The original idea comes from [Ironsworn](https://ironswornrpg.com)'s vows. `Goal
+
+### Filling the Progress Clock
+
+When the progress clock fills up the players should decide, based on recent events, what makes the most sense in the storyline:
+Add a new `supporting detail` - the `disposition` of this detail should generally start at `neutral`. Note `supporting details` should all be specific, unique details that tie back into the fiction, so this is a good chance to promote NPCs to recurring characters, or lean into interesting bits of the story.
+Upgrade the `disposition` of an existing `supporting detail` (moving from `negative` to `neutral` to `positive`).
+Remove or replace an existing neutral `supporting detail`. If all the spaces that make sense are taken or positive, consider whether recent events might have completely supplanted an existing `supporting detail`. This could be a good fit for story elements that have served their purpose or didn't really get much play.
+Create a new immediate opportunity - the GM exposes an opportunity and offers the players with a positive `supporting detail` _if_ they complete a mission to obtain it.
+
+> #### Example
+> 
+> The party is back at spacedock and has gathered enough progress to fill the progress clock. They collectively decide they would like to get the target off their back. They choose to improve the authority `supporting detail` “falsified spacecraft registration” (-), and rewrite it as “overhauled ship hides stolen past” (0). They haven't completely gotten rid of the mysterious origin of the ship, but it won’t put them in such disadvantageous `positions` any more.
+
+## Other Uses for Key Concepts and Supporting Details
+
+> #### Designer Note
+> 
+> Certain games may pick and choose between the following options - while they do not add a large amount of complexity, using the full set of rules grants the players many additional options to see them through tough spots. 
+
+### Supporting Details Grant Conditional Position
+
+When dealing with a situation that directly relates to a `supporting detail`, the `disposition` of the supporting detail (positive or negative) may be used to help determine the `position` of the roll. 
+
+### Key Concepts Grant Conditional Position
+
+When dealing with a situation that might benefit from a positive `key concept`, the `disposition` of the `key concept` may be used to help determine the `position` of the roll. Since `key concepts` are generic to category descriptors, this gives a much broader benefit than individual `supporting details`. 
+
+The same works in reverse - allowing a `key concept` to become broadly negative can make the lives of the party more desperate.
```

Comment:
suggestion: I thinks those two things could be wrapped inside the same category like "How the party sheet affects the fiction" or something (bad example but you get the idea 😅 )